CRM/Financial/DAO/PaymentProcessorType.php

```diff
@@ -207,8 +208,9 @@ public static function &fields() {
         'title' => [
           'name' => 'title',
           'type' => CRM_Utils_Type::T_STRING,
-          'title' => ts('Payment Processor Title'),
-          'description' => ts('Payment Processor Name.'),
+          'title' => ts('Payment Processor Type Title'),
+          'description' => ts('Payment Processor Type Title.'),
+          'required' => TRUE,
```

Comment:
Is this required? For processor it isn't maybe it is for type?

Reply:
Wel that is the DAO for the Type ;)